src/composables/aeSdk.ts

```diff
@@ -116,7 +116,7 @@ export function useAeSdk({ store }: IDefaultComposableOptions) {
       onDisconnect(aeppId: string) {
         delete aeppInfo[aeppId];
       },
-      async onSubscription(aeppId: string) {
+      async onSubscription(aeppId: string, params: any, origin: string) {
```

Comment:
Those new arguments are not used within the scope of the function.

Reply:
`origin` is used, and to get `origin` we need to have `params` too.